pkg/interfaces/contracts/pool-utils/IManagedPool.sol

```diff
@@ -221,6 +228,12 @@ interface IManagedPool is IBasePool {
      * @dev This can be called by anyone to collect accrued AUM fees - and will be called automatically
      * whenever the supply changes (e.g., joins and exits, add and remove token), and before the fee
      * percentage is changed by the manager, to prevent fees from being applied retroactively.
+     *
+     * Correct behavior depends on the token balances from the Vault, which may be out of sync with the state of
+     * the pool during execution of a Vault hook.
+     *
+     * See https://forum.balancer.fi/t/reentrancy-vulnerability-scope-expanded/4345 for reference.
```

Comment:
I'm not sure I understand this comment, what is the issue here?

Reply:
I pushed a clarifying comment (since it's the supply that's relevant, not the token balances)